docs/general/attribute-naming.md

```diff
@@ -31,6 +31,8 @@ particular programming language or wire format._
 
 Names SHOULD follow these rules:
 
+- Names SHOULD be lowercase.
```

Comment:
Minor, might worth considering something formal https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/api.md#instrument-name-syntax

Reply:
Not blocking this PR.

Reply:
good point. I will look into it, and submit a follow up PR.